### ISSUE TITLE
keyBoardControl not working in version 2.6.3

### ISSUE THREAD

**Issue by pejman1** [YIZHUANG/react-multi-carousel#276]
after updating to version 2.6.3 keyboard control does not work 
downgrade to the version 2.6.2 will solve the problem 

windows 10 - react version 17

**Comment by pejman1** [comment]
https://github.com/YIZHUANG

**Comment by YIZHUANG** [comment]
@tegaadigu could https://github.com/YIZHUANG/react-multi-carousel/pull/252 have caused the issue?

**Comment by tegaadigu** [comment]
@YIZHUANG yes and i did create a fast follow fix for this: 

Please review: 

https://github.com/YIZHUANG/react-multi-carousel/pull/260

**Comment by YIZHUANG** [comment]
> @YIZHUANG yes and i did create a fast follow fix for this:
> 
> Please review:
> 
> #260

Thank you, is it possible to make it work for most of the users for now by removing the check for isChildOfCarousel? Cause I don't think I have the time to review the PR yet

**Comment by tegaadigu** [comment]
@YIZHUANG  yes that logic has been removed in the above PR - it doesnt check fo childCarousel anymore. 


https://github.com/YIZHUANG/react-multi-carousel/pull/260

**Comment by pejman1** [comment]
The problem has not fixed yet. I have just installed version 2.8.0 and keyBoardControl doesn't work for me but version 2.6.1 is okay.

**Comment by YIZHUANG** [comment]
> The problem has not fixed yet. I have just installed version 2.8.0 and keyBoardControl doesn't work for me but version 2.6.1 is okay.

Can you open a PR for this?

**Comment by Sammii** [comment]
> > The problem has not fixed yet. I have just installed version 2.8.0 and keyBoardControl doesn't work for me but version 2.6.1 is okay.
> 
> Can you open a PR for this?

Hi @YIZHUANG , I see above that @tegaadigu  already opened a PR to fix this **9 months ago**. Is there some reason it's still not merged?

**Comment by tegaadigu** [comment]
@YIZHUANG i had a pr open to resolve this since may of 2021 - please merge when you can https://github.com/YIZHUANG/react-multi-carousel/pull/260

**Comment by pejman1** [comment]
@YIZHUANG  still the problem has not fixed after  year. since version 2.6.1 !!!
